app/models/holiday.rb

```diff
@@ -66,33 +59,13 @@ def weekend?(date)
     end
 
     def holiday?(date)
-      weekend?(date) ||
-        regular_holiday?(date) ||
-        irregular_holiday?(date)
+      weekend?(date) || irregular_holiday?(date)
```

Comment:
Detail: Ich würde da das Naming noch anpassen, da es jetzt ja keine regular und irregular holidays mehr gibt. Z.B. diese Methode in `non_working_day?` und `irregular_holiday?` in `holiday?`

Reply:
:+1:

Reply:
Geändert in commit https://github.com/puzzle/puzzletime/pull/47/commits/35414c6428b55a4e992ae66fd3b9b18399a58899